includes/template-tags.php

```diff
@@ -61,3 +61,67 @@ function adjust_brightness( $hex, $steps ) {
 	return $return;
 
 }
+
+/**
+ * An svg template helper.
+ *
+ * @param  string  $name   The SVG Name.
+ * @param  array   $opts   Optional. The SVG options such as Role, Class, Label and Hidden.
+ * @param  boolean $output Either return or output the SVG Icon string.
+ * @return html
+ */
+function svg_icon( $name = '', $opts = array(), $output = false ) {
+	if ( empty( $name ) ) {
+		return '';
+	}
+
+	if ( is_customize_preview() ) {
+		$href = TENUP_SCAFFOLD_TEMPLATE_URL . '/dist/svg-sprite.svg#' . $name;
+	} else {
+		$href = '#' . $name;
+	}
+
+	$role = empty( $opts['role'] ) ? 'presentation' : $opts['role'];
+
+	// Fix 'image'.
+	if ( 'image' === $role ) {
+		$role = 'img';
+	}
+
+	$class = empty( $opts['class'] ) ? 'dmsb-icon dmsb-icon-' . $name : 'dmsb-icon ' . $opts['class'];
```

Comment:
@colorful-tones namespaced for northeastern still

Reply:
I'm going to resolve most of these conversations, because most of them have been addressed for some time.